pma.sh

```diff
@@ -24,6 +24,6 @@ fi
 sudo bash $CMD phpmyadmin.test $(pwd)/phpmyadmin 80 443 7.3
 
 echo "Installing dependencies for phpMyAdmin"
-cd phpmyadmin && composer update --no-dev
+cd phpmyadmin && composer update --no-dev && yarn
```

Comment:
`yarn install --production`

https://github.com/phpmyadmin/phpmyadmin/issues/16178

Reply:
> `yarn install --production`
> 
> [phpmyadmin/phpmyadmin#16178](https://github.com/phpmyadmin/phpmyadmin/issues/16178)

Hello William, would you open a pull request?

Reply:
Sure, here it is 
https://github.com/grrnikos/pma/pull/33